io_scene_nif/modules/nif_export/property/texture/__init__.py

```diff
@@ -91,6 +115,33 @@ def get_uv_node(self, b_texture_node):
             raise util_math.NifError(f"Unsupported vector input for {b_texture_node.name} in material '{self.b_mat.name}''.\n"
                                      f"Expected 'UV Map' or 'Texture Coordinate' nodes")
 
+    def get_global_uv_transform_clip(self, b_texture_node):
+        #get the values from the nodes, find the nodes by name, or search back in the node tree
+        x_scale = y_scale = x_offset = y_offset = clamp_x = clamp_y = None
+        #first check if there are any of the preset name - much more time efficient
+        try:
+            combine_node = b_texture_node.id_data.nodes["Combine UV0"]
+            if not isinstance(combine_node, bpy.types.ShaderNodeCombineXYZ):
+                combine_node = self.get_input_node_of_type(b_texture_node.inputs[0], bpy.types.ShaderNodeCombineXYZ)
+        except:
+            #if there is a combine node, it does not have the standard name
+            combine_node = self.get_input_node_of_type(b_texture_node.inputs[0], bpy.types.ShaderNodeCombineXYZ)
```

Comment:
Can this still fail.

Reply:
It hasn't, in my experience (but I didn't test much), and I tried to make it so it shouldn't. If it fails to find something, it returns `None`.

Reply:
Cool, something we can keep in the back of our minds.
Ultimately we'll see what happens when this is loose with users.